code/go/0chain.net/blobbercore/handler/grpchandler.go

```diff
@@ -0,0 +1,195 @@
+package handler
+
+import (
+	"context"
+
+	"0chain.net/blobbercore/reference"
+
+	"github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
+
+	"0chain.net/core/common"
+
+	"go.uber.org/zap"
+
+	grpc_recovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
+
+	grpc_zap "github.com/grpc-ecosystem/go-grpc-middleware/logging/zap"
+
+	"github.com/gorilla/mux"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+
+	"google.golang.org/grpc"
+
+	"0chain.net/blobbercore/blobbergrpc"
+	"0chain.net/core/logging"
+)
+
+type blobberGRPCService struct {
+	storageHandler StorageHandler
+	blobbergrpc.UnimplementedBlobberServer
+}
+
```

Comment:
It might look better to group `blobberGRPCService` with its methods and constructor.